python-frontend/src/test/java/org/sonar/python/semantic/v2/SymbolTableBuilderV2Test.java

```diff
@@ -37,13 +37,13 @@ class SymbolTableBuilderV2Test {
 
   @BeforeAll
   static void init() {
-    var context = TestPythonVisitorRunner.createContext(new File("src/test/resources/semantic/v2/script.py"));
+    var context = TestPythonVisitorRunner.createContext(new File("src/test/resources/semantic/v2/symbol_table.py"));
```

Comment:
I think your branch and mine have a conflict on how we dealt with this issue. I decided to inline the code in a text-block, which I feel makes it clearer the snippet should not be changed. Wdyt of this approach?